migtests/tests/oracle/partitions/partition_schema.sql

```diff
@@ -51,25 +51,33 @@ PARTITION BY RANGE (order_date)
 ALTER TABLE ORDERS_INTERVAL_PARTITION ADD (CONSTRAINT ordid_orddate_pk PRIMARY KEY (order_id, order_date));
 
 --HASH PARTITIONS
-DROP TABLESPACE tbs1 including contents;
-DROP TABLESPACE tbs2 including contents;
-DROP TABLESPACE tbs3 including contents;
-DROP TABLESPACE tbs4 including contents;
+-- DROP TABLESPACE tbs1 including contents;
+-- DROP TABLESPACE tbs2 including contents;
+-- DROP TABLESPACE tbs3 including contents;
+-- DROP TABLESPACE tbs4 including contents;
 
-CREATE TABLESPACE tbs1 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
-CREATE TABLESPACE tbs2 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
-CREATE TABLESPACE tbs3 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
-CREATE TABLESPACE tbs4 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
+-- CREATE TABLESPACE tbs1 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
+-- CREATE TABLESPACE tbs2 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
+-- CREATE TABLESPACE tbs3 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
+-- CREATE TABLESPACE tbs4 DATAFILE SIZE 1G AUTOEXTEND ON MAXSIZE 10G;
```

Comment:
Why is this commented? Do we plan to uncomment it in future? If yes, file a ticket and add a link in a comment. Else, simply get rid of this.